### ISSUE TITLE
Title showing wrong if Rank Math SEO activated

### ISSUE THREAD

**Issue by milanhirapra** [buddyboss/buddyboss-platform#2696]
**Describe the bug**
Activate the Rank Math SEO plugin and open the following pages User Timeline, User Connection, User Groups, then it will only show the username and site title.

**To Reproduce**
1. Go to website > user profile dropdown
2. Click on Timeline
3. Title showing wrong

**Expected behavior**
The title should display according to the page.

**Screenshots**
https://vimeo.com/580690958/b81bfb805a

**Environment**
Which site did you replicate this issue on?
1) Customer: https://wordpressninja.net/members/
2) My Dev site: https://dev-milan.flywheelsites.com/
3) My local machine 

**Support ticket links**
https://secure.helpscout.net/conversation/1575649197/154268/

**Jira issue** : [PROD-778]

[PROD-778]: https://buddyboss.atlassian.net/browse/PROD-778?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by emmanoctaviano** [comment]
https://secure.helpscout.net/conversation/1579408360/154940

**Comment by laflier** [comment]
I think I've found the problem.

In bp-core/compatibility/bp-rankmath-plugin-helpers.php, there is a function starting on line 25 that sets BuddyPress page titles to the user's first_name (or nickname) followed by a dash and the site title. 

I'm not sure how this function improves compatibility with Rank Math, but it's clearly what is causing the unexpected behavior. In the bug I identified in the support ticket, Every BuddyPress page has the title "{first_name} - {site_name}," which is exactly what this function does.

**Confirmation**
Comment out the body of the function and return the unaltered $title. Output is as expected.

**Source code**
https://github.com/buddyboss/buddyboss-platform/blob/release/src/bp-core/compatibility/bp-rankmath-plugin-helpers.php
